clash-lib/src/Clash/Netlist/Util.hs

```diff
@@ -1244,25 +1244,15 @@ mkInput pM = case pM of
         _ ->  return ([(pN,hwty)],[],Identifier pN Nothing,pN)
 
 -- | Create a Vector chain for a list of 'Identifier's
-mkVectorChain :: Int
-              -> HWType
-              -> [Expr]
-              -> Expr
-mkVectorChain _ elTy []      = DataCon (Vector 0 elTy) VecAppend []
-mkVectorChain _ elTy [e]     = DataCon (Vector 1 elTy) VecAppend
-                                [e]
-mkVectorChain sz elTy (e:es) = DataCon (Vector sz elTy) VecAppend
-                                [ e
-                                , mkVectorChain (sz-1) elTy es
-                                ]
+mkVectorChain :: Int -> HWType -> [Expr] -> Expr
+mkVectorChain _ elTy [] = DataCon (Vector 0 elTy) VecAppend []
+mkVectorChain _ elTy [e] = DataCon (Vector 1 elTy) VecAppend [e]
+mkVectorChain sz elTy (e:es) =
+  DataCon (Vector sz elTy) VecAppend [e, mkVectorChain (sz-1) elTy es]
```

Comment:
Not a fan of these changes, they neither clarify (e.g. more comments), nor change the behavior; so why not leave as is?

Reply:
Fair enough. I just thought they were formatted very far from what the styleguide says.

Reply:
And I usually don't mind cleanup-commits, my IDE lets me click through them easily. I'll revert the change though.